tests/python/relay/test_analysis_pipeline.py

```diff
@@ -0,0 +1,69 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import numpy as np
+import tvm
+import tvm.testing
+from tvm import relay
+from tvm.relay import transform
+from tvm.contrib import graph_runtime
+from tvm.relay.analysis import pipeline_graph
+
+
+def run_module(mod, ctx, dname, data):
+    with relay.build_config(opt_level=3):
+        graph, lib, params = relay.build(mod, "llvm")
+
+    m = graph_runtime.create(graph, lib, ctx)
+    m.set_input(dname, data)
+    m.set_input(**params)
+    m.run()
+    output = m.get_output(0).asnumpy()
+    return output
+
+
+def get_network():
+    dshape = (3, 3)
+    mvalue = np.full((1), 4).astype("float32")
+    mmv = relay.Constant(tvm.nd.array(mvalue))
+    mv = relay.Constant(tvm.nd.array(mvalue))
+    mv2 = relay.Constant(tvm.nd.array(mvalue))
+    mv3 = relay.Constant(tvm.nd.array(mvalue))
+    data = relay.var("data", relay.TensorType(dshape, "float32"))
+    net = relay.multiply(data, mv)
+    net = relay.add(net, mv2)
+    net = relay.subtract(net, mv3)
+    net = relay.add(net, mv3)
+    func = relay.Function([data], net)
+    mod = tvm.IRModule.from_expr(func)
+    return mod, dshape
+
+
+mod, dshape = get_network()
+pl = [1, 2]
```

Comment:
Should we test also more exotic cases like [0 3 5]? Would this still work (I think so, but better add a test)